loadtest/user/userentity/user.go

```diff
@@ -6,10 +6,13 @@ package userentity
 import (
 	"errors"
 	"net/http"
+	"os"
 	"time"
 
-	"github.com/gocolly/colly/v2"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/store"
+	perf "github.com/mattermost/mattermost-load-test-ng/performance"
```

Comment:
Do we really need to use named import here?

Reply:
It was mainly to avoid the quite lengthy `*performance.UserEntityMetrics`. 
If you think the named import is more annoying than the excessive length I can remove it :stuck_out_tongue: 
Also we could still name the package `perf`, just personally liked the whole word better.

Reply:
I see, in some places we used in full length e.g in `api/server.go` as `*performance.Metrics`, to maintain consistency maybe we can rename the package or rename here, I'm fine with both 😅

Reply:
Great eye @isacikgoz !